### PR TITLE
Update tokens.json

### PR DESCRIPTION
**Pull request by puffalortis**
the tokens are on elliottkelly on eosio, im just not sure if i got the account name part of this correct.  please email me at tucsonset@gmail.com if need arises.

## How to add token?

- [ ] Add metadata to [`tokens.json`](tokens.json)

```json
{
    "name": "<NAME>",
    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "symbol": "<SYMBOL>",
    "account": "<CONTRACT NAME>",
    "chain": "eos"
}
```

- [ ] Add token logo `*.png` to [`./logos`](./logos) folder
- [ ] Update `README.md` with **Symbol** & **Contract Name**

### CODE REVIEW

File: tokens.json [hunk 1]
```diff
@@ -2314,5 +2314,13 @@
         "symbol": "LIUAIEV",
         "account": "token.list",
         "chain": "eos"
-    }
+    },
+    {
+    "name": "down",
+    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/downcoinlogo.png",
```
Comment (DenisCarriere) [inline tokens.json:2320]: Need to include the png image to this repo

File: tokens.json [hunk 2]
```diff
@@ -2314,5 +2314,13 @@
...
+    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<downcoinlogo.png",
```
Comment (DenisCarriere) [inline tokens.json:2321]: typo `<downcoinlogo.png`

**Review by DenisCarriere — CHANGES_REQUESTED** [review]

File: tokens.json (hunk 2 above)
Comment (puffalortis) [reply tokens.json:2321]: +    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/downcoinlogo.png",
+    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/
downcoinlogo
downcoinlogo.png",

**Review by puffalortis — COMMENTED** [review]
downcoinlogo

**Pull request closed without merging**